db/migrate/20170425154145_add_sti_to_middlewares.rb

```diff
@@ -0,0 +1,89 @@
+class AddStiToMiddlewares < ActiveRecord::Migration[5.0]
+  class MiddlewareDatasource < ApplicationRecord
+    self.inheritance_column = :_type_disabled
+    self.table_name = 'middleware_datasources'
```

Comment:
I'm not sure that's the case, actually, because the full class name is `AddStiToMiddlewares::MiddlewareDatasource`, which I think it will be parsed by AR as `add_sti_to_middlewares_middleware_datasources`, and not just `middleware_datasources`.